pyanaconda/users.py

```diff
@@ -161,6 +164,20 @@ def validatePassword(pw, user="root", settings=None):
 
     return (valid, strength, message)
 
+def check_name(in_name):
+    ign_dir = {}
+    for inc in os.listdir("/") + ["root", "home", "daemon"]:
+        ign_dir.update({inc: "Cannot set user name, reserved for system: "})
```

Comment:
If adding a key-value pair to the dict, use the ``some_dict[key] = value`` syntax not the ``update`` method.